src/css/styles.css

```diff
@@ -144,7 +186,25 @@ a:hover {
     font-size: 18px;
   }
 
+  .section {
+    padding: 70px 85px;
+  }
+
   .subhead {
     font-size: 22px;
   }
+
+  .text-group {
+    padding: 0 70px;
+  }
+}
+@media only screen and (max-width: 767px) {
```

Comment:
We should probably make these variables somewhere

Reply:
@matthew-gerstman what is best way to leave a todo. I would guess we can import from the `responsive.css` file, but i haven't taken stab at that yet.